model/status.go

```diff
@@ -27,6 +27,31 @@ type Status struct {
 	ActiveChannel  string `json:"-" db:"-"`
 }
 
+type UserStatusClusterMessage struct {
+	*Status
+	ActiveChannel string `json:"active_channel"`
+}
```

Comment:
If anyone has a better way of doing this in Go, please let me know. Issue I'm getting around is `ActiveChannel` is omitted from the json in the regular Status struct. I want it to only be included when sending cluster messages

Reply:
The other way I would think of doing this is adding omitempty and just blanking it out in the ToJson method. Then have another tojson method that doesn't blank it.

Reply:
I considered doing that but then I'd have to copy/create a new status struct each time the ToJson() method is called so as to not clobber the ActiveChannel field of the original status struct. So I opted for this instead, where I'm still creating new objects but only on cluster send not on every ToJson() for statuses. 0/5 as well, just thought it might be better to copy only on cluster send

Reply:
Would this not work?
```go
func (o *Status) ToJson() string {
    tmp := o.ActiveChannel
    o.ActiveChannel = ""
    b, _ := json.Marshal(o)
    o.ActiveChannel = tmp
    return string(b)
 }
```
Then have a second method that doesn't hide the active channel field.

Reply:
Oh yeah that'll probably work, let me try it